tests/unit/replicator_tests.py

```diff
@@ -61,8 +61,22 @@ def tearDown(self):
         self.target_db.delete()
         del self.test_target_dbname
         del self.target_db
-        while self.replication_ids:
-            self.replicator.stop_replication(self.replication_ids.pop())
+
+        for rep_id in self.replication_ids:
+            max_retry = 5
+            while 1:
```

Comment:
k, will change. I used `while 1` as `True` is not a keyword in 2.x, just a global const. But you'd be crazy to set `True = False` anyway.